openpower/package/libflash/libflash.mk

```diff
@@ -5,8 +5,10 @@
 ################################################################################
 
 LIBFLASH_VERSION = v5.7-76-g830fc9a0ed0a
-
 LIBFLASH_SITE = $(call github,open-power,skiboot,$(LIBFLASH_VERSION))
+HOST_LIBFLASH_VERSION = 830fc9a0ed0a4066129cfafd1447dc40a66b8700
```

Comment:
This actually equals the `v5.7-76-g830fc9a0ed0a` version above. Is there a reason to have separate versions?